src/config/session-management.md

```diff
@@ -30,16 +30,5 @@ also be one of the mechanisms for rootless [Xorg](./graphical-session/xorg.md).
 To make use of its features, install the `elogind` package and make sure the
 [system D-Bus](#d-bus) is enabled. You might need to log out and in again.
 
-### Troubleshooting
-
 If you're having any issues with `elogind`, [enable](./services/index.md) its
 service, as waiting for a D-Bus activation can lead to issues.
```

Comment:
Still leave a comment leading back to power management? Like `elogind also does [power management](./power-management.md).`

Reply:
More importantly, mention that some considerations can be necessary when installing elogind, which I hope leads to people reading about the potential issues.

Reply:
i've just pushed some changes; let me know what you think.

Reply:
Looks good. Thanks c: